File: trip_chill_backend_test/RegularExpression.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text.RegularExpressions;
using System.Threading.Tasks;

namespace trip_chill_backend_test
{
    public class RegularExpression
    {
        public bool validateEmail(string email)
        {
            return Regex.IsMatch(email, @"^([\w\.\-]+)@([\w\-]+)((\.(\w){2,3})+)$");
        }

        public bool validatePhone(string Phone)
        {
            return Regex.IsMatch(Phone, @"^09[0-9]{8}$");
        }
    }
}

File: trip_chill_view/wwwroot/js/RegularExpression.js
function validateEmail(email) {
    var re = /^\w+([-+.']\w+)*@\w+([-.]\w+)*\.\w+([-.]\w+)*$/;

    return re.test(email);
}

function validatePhone(phone) {
    var re = /^09\d{8}$/;

    return re.test(phone);
}